src/Stratis.Bitcoin/Consensus/ConsensusRuleEngine.cs

```diff
@@ -118,6 +118,11 @@ public virtual Task Initialize()
             return Task.CompletedTask;
         }
 
+        /// <inheritdoc />
+        public virtual void Dispose()
+        {
+        }
```

Comment:
eh?

Reply:
To satisfy the interface and create an optional overload